src/components/App.tsx

```diff
@@ -23,11 +23,17 @@ function App() {
 			console.log('Handling teleop data:', teleop);
 			// You can add more logic to process the teleop data
 		}
+		function teamData(teamData: IMatch): void {
+			// Implement the team data handling logic here
+			console.log('Handling team data:', teamData);
+			// You can add more logic to process the team data
```

Comment:
Here we can import the gearscout service and call `gearscoutService.submitMatch(user, match)`